content/zh/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -120,7 +120,7 @@ Containers are becoming popular because they have many benefits. Some of the con
   容器镜像构建和部署。
 * 关注开发与运维的分离：在构建/发布时而不是在部署时创建应用程序容器镜像，
   从而将应用程序与基础架构分离。
-* 可观察性不仅可以显示操作系统级别的信息和指标，还可以显示应用程序的运行状况和其他指标信号。
+* 可观察性：不仅可以显示操作系统级别的信息和指标，还可以显示应用程序的运行状况和其他指标信号。
```

Comment:
This is consistent with the above English version, maybe you can also submit a PR in the English version?

It seems that the English version is also incorrectly formatted.